### ISSUE TITLE
KeyError `task_name`

### ISSUE THREAD

**Issue by zanussbaum** [xlang-ai/instructor-embedding#101]
I get a [key error](https://instructor-embedding.github.io/) when using the data downloaded from [here](https://instructor-embedding.github.io/). Has the key been changed to `task_id` or am I using the wrong data? I don't see `task_name` anywhere in the dictionaries

**Comment by hongjin-su** [comment]
Yes, I fixed the README and train.py. Thanks a lot for the reminder!

**Comment by zanussbaum** [comment]
No problem! Do you have a mapping from `task_name` to `task_id`? It would be handy to know which is which!

**Comment by srikanthmalla** [comment]
medi-data link provided in readme downloads datasets with task_name. @hongjin-su Do you have the right dataset that is described in the readme which says it should have task-id?
